.gitmodules

```diff
@@ -34,3 +34,6 @@
 	url = https://github.com/coreboot/intel-microcode.git
 	update = none
 	ignore = dirty
+[submodule "3rdparty/ffs"]
+	path = 3rdparty/ffs
+	url = ../ffs.git
```

Comment:
@piotr-kleins this link should be changed to full https link, otherwise it will fail in CI

Reply:
Fixed